scripts/apu_fw_updater_opnsense.sh

```diff
@@ -17,7 +17,7 @@
 TYPE="apu2"
 
 # Version of firmware
-VERSION="4.15.0.3"
+VERSION="4.17.0.3"
```

Comment:
Actually v24.05.00.01 is the latest (Dasharo branded already):

https://dl.3mdeb.com/open-source-firmware/Dasharo/pcengines_apu2/v24.05.00.01/

The latest non-Dasharo firmware is v4.19.0.1: 
https://dl.3mdeb.com/open-source-firmware/pcengines/apu2/apu2_v4.19.0.1.rom

Reply:
https://dl.3mdeb.com/open-source-firmware/Dasharo/pcengines_apu2/v24.05.00.01/ only appears to contain signatures/checksums rather than the images themselves - are the actual ROMs available for download?

(I've got just a single APU2e and so I can't justify paying for annual support - and indeed I've built my own SeaBIOS & EFI firmware images from the Open Source repo.  I've held off from applying either, though, as I'm not sure what the recovery process is if the host gets bricked in the process.  At the very least, it'd be helpful to be able to perform a byte-by-byte comparison between the build artefact and the official image, to see whether anything bar possibly date-stamps has changed between the two…)

Reply:
https://shop.3mdeb.com/shop/dasharo-pro-package/1year-desktop/

Reply:
Ahh my bad. Forgot the Dasharo-branded firmware is not public and only available to subscribers. Then v4.19.0.1 is the latest

Reply:
I realised that v4.19.0.1 was the latest, but [the release page](https://pcengines.github.io/) recommends to use v4.17.0.x. Is the release page out of date?

![Screenshot 2025-01-28 at 14 45 38](https://github.com/user-attachments/assets/b9e13332-0572-4e73-b4cb-9885580e9503)

Reply:
Probably someone forgot to update the recommended version.

Reply:
> Ahh my bad. Forgot the Dasharo-branded firmware is not public and only available to subscribers. Then v4.19.0.1 is the latest

you can share the branded releases here, though 😏